packages/iov-bcp-types/src/bcp.ts

```diff
@@ -91,13 +91,25 @@ export interface BcpValueNameQuery {
   readonly name: string;
 }
 
-export type BcpAccountQuery = BcpAddressQuery | BcpValueNameQuery;
+export interface BcpPubkeyQuery {
+  readonly publicKey: PublicKeyBundle;
```

Comment:
Can we try to use "pubkey" as a single work? This makes so much stuff simpler and shorter in the crypto world

Reply:
We should unified this concept, because we are using multiple names in the same repo

Reply:
As discussed in Slack, we make "pubkey" an atomic word and use it at least in all new code and try to convert old code step by step